### ISSUE TITLE
Loss Functions

### ISSUE THREAD

**Issue by mrm-196** [facebookresearch/deepcluster#64]
Hi,
I wonder during the training how the CrossEntropy loss and clustering loss are expected to change per epoch? Is it normal for the clustering loss to be increasing? Is that generalizable to imbalanced datasets?
Thanks for your very interesting work.

**Comment by mathildecaron31** [comment]
Hi, thank you for your interest. The cross entropy and the clustering loss should both decrease during training
